packages/styleguide/stories/Layout.stories.js

```diff
@@ -28,79 +28,79 @@ const Test = ({ color, children }) => {
 
 const KitchenSinkColumns = ({ alt }) => (
```

Comment:
Oh, I just noticed this is unused...

```suggestion
const KitchenSinkColumns = () => (
```